crates/services/txpool/src/txpool.rs

```diff
@@ -252,7 +252,7 @@ where
         self.insert_inner(tx, &view)
     }
 
-    #[tracing::instrument(level = "info", skip_all, fields(tx_id = %tx.id()), ret, err)]
+    #[tracing::instrument(level = "debug", skip_all, fields(tx_id = %tx.id()), ret, err)]
```

Comment:
Should this remain as debug?

Reply:
Yep, because it logs each inserted and removed transactions